_data/sites.json

```
@@ -10061,6 +10061,16 @@
         ]
     },
 
+    {
+        "name": "Smule",
+        "url": "https://smule.zendesk.com/hc/en-us/requests/new",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
Since this requires opening a ticket and waiting for the support to act, I think it is better off as hard